platform/cli/usage-guides/new-service/java/aws/README.md

```diff
@@ -0,0 +1,344 @@
+#### 3.1.5 Connect the webhooks
+
+### 3.2 Deploy Secrets Manager 
+In this section, we will be deploying the KMS keys needed for our secrets.
+
+
+TODO:
+#### 3.2.1 Create the inputs for the Secrets Manager 
+#### 3.2.2 Create the Secrets Manager service
+#### 3.2.3 Generate the Terragrunt files for the Secrets Manager service
+#### 3.2.4 Deploy the Secrets Manager service
+#### 3.2.5 Connect the webhooks
+
+### 3.3 Deploy IAM roles 
+In this section, we will be deploying the IAM roles needed for our platform and java application. 
+
+TODO:
+#### 3.3.1 Create the inputs for the IAM roles
+#### 3.3.2 Create the IAM roles service
+#### 3.3.3 Generate the Terragrunt files for the IAM roles service
+#### 3.3.4 Deploy the IAM roles service
+#### 3.3.5 Connect the webhooks
+
+### 3.4 Deploy ECR Repository
+In this section, we will be deploying the necessary infrastructure to store our java image built. 
+
+current: `arn:aws:ecr:us-east-2:538234414982:repository/launch-api`
+
+TODO:
+#### 3.4.1 Create the inputs for the ECR Repository 
+#### 3.4.2 Create the ECR Repository service
+#### 3.4.3 Generate the Terragrunt files for the ECR Repository service
+#### 3.4.4 Deploy the ECR Repository service
+#### 3.4.5 Connect the webhooks
+
+### 3.5 Deploy ECS Platform
+In this section, we will be deploying the ECS platform that our application will be deployed onto. 
+
+#### 3.5.1 Create the inputs for the ECS Platform
+This guide has provided basic inputs to be used with the services we are deploying. However, we cannot use these right out of the box and we need to quickly update some paths within our `.launch_config` file.
+
+The launch config for the platform in this guide is at the following:
+- [./example_files/platform/.launch_config](example_files/platform/.launch_config)
+
+Open this file and update the `properties_file` key with the exact path from your system to the input files to be used, and then save it.
+
+We are going to be using the following inputs for our `.launch_config` files. 
+- [Service `properties file ./example_files/platform/service.sandbox.us-east-2.tfvars](./example_files/platform/service.sandbox.us-east-2.tfvars)
+- [Pipeline `properties file` ./example_files/platform/pipeline.root.us-east-2.tfvars](./example_files/platform/pipeline.root.us-east-2.tfvars)
+- [Webhooks `properties file` ./example_files/platform/webhooks.root.us-east-2.tfvars](./example_files/platform/webhooks.root.us-east-2.tfvars)
+
+<p align="center">
+  <img src="./pictures/launch_config-paths-platform.png" /> 
+</p>
+
+#### 3.5.2 Create the ECS Platform service
+We are now going to create the ECS platform properties repository.
+Replace the path in the `--in-file` argument to the exact path of the `.launch_config` file saved in the previous section. 
+```
+~ $ launch service create --name launch-demo-ecs-platform --in-file /workspaces/workplace/common-platform-documentation/platform/cli/usage-guides/new-service/java/aws/example_files/platform/.launch_config
```

Comment:
I've seen plenty of documentation that includes the $ or # character to differentiate a line entered into the prompt from its output, but including the path (~) seems irrelevant here since there are no relative paths being used.